docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
       OCIS_URL: ${OCIS_URL:-https://host.docker.internal:9200}
       STORAGE_HOME_DRIVER: ${STORAGE_HOME_DRIVER:-ocis}
       STORAGE_USERS_DRIVER: ${STORAGE_USERS_DRIVER:-ocis}
+      STORAGE_LDAP_CACERT: ${STORAGE_LDAP_CACERT:-/web/ldap.crt}
```

Comment:
is this really needed in that case? We have the LDAP cert related variables all set together in the dockerfile: https://github.com/owncloud/ocis/blob/3afc5992e63e7ce12e3c213476f752a57643793a/ocis/docker/Dockerfile.linux.amd64#L33-L35

Reply:
Maybe the image wasn't up-to-date then, because on Friday it was trying to store the `ldap.crt` in the home directory of the
user in drone.